_i18n/de.yml

```
@@ -0,0 +1,90 @@
+meta:
+  description: Spielentwicklung in 2D & 3D, für platt­form­über­grei­fende Projekte sowie VR Ideen!
+header:
+  features: Funktionen
+  showcase: Galerie
+  blog: Blog
+  community: Vernetzen
```

```suggestion
  community: Community
```
Probably best to reverse that like @mhilbrunner said

---

Done 👍🏻